<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>load_dialog</class>
 <widget class="QDialog" name="load_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wczytywanie danych</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="load_operations_text"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="load_operations_label">
     <property name="text">
      <string>Operacje</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="load_operations_button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="load_categories_text"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="load_categories_label">
     <property name="text">
      <string>Kategorie</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="load_products_button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="load_categories_button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="load_products_text"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="load_products_label">
     <property name="text">
      <string>Produkty</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="load_button">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
